### PR TITLE
Add some instructions for the reviewer

### PR DESCRIPTION
**Pull request by marijnschilling**
I hope this will get new developers up to speeds with what to when reviewing!

### CODE REVIEW

File: GIT_AND_GITHUB.md [hunk 1]
```diff
@@ -75,7 +75,29 @@ submitter and go through the pull request together.
 Be warned, though: By doing this, you will both need to explain your thoughts and
 discuss other alternatives. Of course, this means you are putting yourself at risk
 of sharing your knowledge and/or learning some new stuff, so please be careful not
-to end up being even more awesome than you are.
+to end up being even more awesome than you already are.
```
Comment (elland) [inline GIT_AND_GITHUB.md:78]: 😍

File: GIT_AND_GITHUB.md [hunk 2]
```diff
@@ -75,7 +75,29 @@ submitter and go through the pull request together.
...
+
+As a reviewer your job is to be the extra pair of eyes, so the code will get twice 
+as good. You will look at thing like code style, structure, discuss things you might
```
Comment (elland) [inline GIT_AND_GITHUB.md]: What about: `You'll look at things such as code style and structure. You'll discuss things that you might've done differently, point out possible more elegant solutions based on something you've learned, and so on. The goal here is to make the whole team better in the process.` Some grammar corrections etc. 😄

File: GIT_AND_GITHUB.md [hunk 3]
```diff
@@ -75,7 +75,29 @@ submitter and go through the pull request together.
...
 of sharing your knowledge and/or learning some new stuff, so please be careful not
-to end up being even more awesome than you are.
+to end up being even more awesome than you already are.
+
+As a reviewer your job is to be the extra pair of eyes, so the code will get twice 
+as good. You will look at thing like code style, structure, discuss things you might 
+do differently, point out what you've recently learned or applied. The goal is to 
+make the whole team become better at what we do!
+
+Keep in mind that you are reviewing what someone worked on really hard, so always be
```
Comment (elland) [inline GIT_AND_GITHUB.md]: `reviewing something that someone else have worked on really hard`

**Comment by 3lvis** [comment]
Great additions, @marijnschilling!

File: GIT_AND_GITHUB.md (hunk 2 above)
Comment (marijnschilling) [inline GIT_AND_GITHUB.md]: I like it! Was struggling a bit with this part.

**Comment by elland** [comment]
💥 nice ideas. 💃

File: GIT_AND_GITHUB.md (hunk 3 above)
Comment (marijnschilling) [inline GIT_AND_GITHUB.md]: I like it! More explicit. But shouldn't it be 'has'?

**Comment by 3lvis** [comment]
Merge it!

**Pull request merged**